Formula/t/templ.rb

```diff
@@ -0,0 +1,28 @@
+class Templ < Formula
+  desc "Language for writing HTML user interfaces in Go"
+  homepage "https://templ.guide"
+  url "https://github.com/a-h/templ/archive/refs/tags/v0.2.793.tar.gz"
+  sha256 "be924c8b359748b0109c18831e1105618f6174a0f40645d5b82a91e7a9a8fffc"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/templ"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/templ version")
+
+    (testpath/"test.templ").write <<~TEMPL
```

Comment:
TEMPL is not a language for syntax highlighting?

```suggestion
    (testpath/"test.templ").write <<~EOS
```

Reply:
No, it can be syntax highlighted. Github supports it for example (eg https://github.com/a-h/templ/blob/main/examples/blog/posts.templ)

Reply:
I see, ,sounds good 👍